src/main/java/org/opentripplanner/graph_builder/module/ned/ElevationModule.java

```diff
@@ -14,57 +16,105 @@
 import org.opentripplanner.graph_builder.annotation.Graphwide;
 import org.opentripplanner.graph_builder.module.GraphBuilderModuleSummary;
 import org.opentripplanner.graph_builder.module.GraphBuilderTaskSummary;
-import org.opentripplanner.graph_builder.module.extra_elevation_data.ElevationPoint;
 import org.opentripplanner.graph_builder.services.GraphBuilderModule;
 import org.opentripplanner.graph_builder.services.ned.ElevationGridCoverageFactory;
 import org.opentripplanner.routing.edgetype.StreetEdge;
 import org.opentripplanner.routing.edgetype.StreetWithElevationEdge;
 import org.opentripplanner.routing.graph.Edge;
 import org.opentripplanner.routing.graph.Graph;
 import org.opentripplanner.routing.graph.Vertex;
+import org.opentripplanner.util.PolylineEncoder;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import javax.media.jai.InterpolationBilinear;
-import java.util.ArrayList;
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.nio.file.Files;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.LinkedList;
 import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.opentripplanner.util.ElevationUtils.computeEllipsoidToGeoidDifference;
 
 /**
- * {@link org.opentripplanner.graph_builder.services.GraphBuilderModule} plugin that applies elevation data to street data that has already
- * been loaded into a (@link Graph}, creating elevation profiles for each Street encountered
- * in the Graph. Depending on the {@link ElevationGridCoverageFactory} specified
- * this could be auto-downloaded and cached National Elevation Dataset (NED) raster data or
- * a GeoTIFF file. The elevation profiles are stored as {@link PackedCoordinateSequence} objects,
- * where each (x,y) pair represents one sample, with the x-coord representing the distance along
- * the edge measured from the start, and the y-coord representing the sampled elevation at that
+ * {@link org.opentripplanner.graph_builder.services.GraphBuilderModule} plugin that applies elevation data to street
+ * data that has already been loaded into a (@link Graph}, creating elevation profiles for each Street encountered
+ * in the Graph. Depending on the {@link ElevationGridCoverageFactory} specified this could be auto-downloaded and
+ * cached National Elevation Dataset (NED) raster data or a GeoTIFF file. The elevation profiles are stored as
```

Comment:
Please clarify this sentence.

Reply:
I'm not sure I understand. What additional clarity is needed here?

Reply:
It doesn't quite read like a sentence. I would tweak like this:
- comma after 'specified',
- 'this could be an auto-downloaded and cached [NED] or a GeoTiFF file' 
    -- OR -- 'this could be auto-downloaded and cached from a [NED] or a GeoTiFF file '